docs/lib/content/configuring-npm/package-json.md

```diff
@@ -339,6 +339,69 @@ not much else.
 
 If `main` is not set, it defaults to `index.js` in the package's root folder.
 
+### exports
+
+The exports field is an object that maps entry points to modules. This field is
+supported by Node.js versions including and higher than 12. It acts as a more
```

Comment:
> higher than 12

see https://npmjs.com/node-exports-info - there’s actually a pretty complex set of feature support and node version matrix.

Reply:
do you think it would be better to take out the "higher than 12" comment since the version of NPM these docs are for only supports Node.js `^18.17.0 || >=20.5.0` anyway?

Reply:
yes - or maybe, "is supported by modern node.js versions"? also this should link to the node documentation of the field, which might be better than having examples here.

Reply:
I've added your suggestions, how's this so far?